### ISSUE TITLE
Add `cypress-axe` to the Cypress tests

### ISSUE THREAD

**Issue by 1Copenut** [1Copenut/c3-eleventy#78]
Make sure you're configuring it like you've done previous ones to use a custom reporter, allow reporting without failing, and all the cool reporting features like Mochawesome.
